### PR TITLE
Allow import data urls

### PR DESCRIPTION
**Pull request by epeach**
Zendesk reports visible in Jira ticket.

Summary: Received reports from users that screens with data-urls could not be imported.

Cause: The length of the data-urls were causing the ajax request to fail.

Fix: Filtered the data-urls out of files sent to the copy-assets ajax request. This means the list of ajax requests do not fail. When the screens then go through and copy the property for the imported screen to the importee screen, the data-url is copied. Because the data-url contains all the information needed to render the image, image is able to render in the importee screen.

## Links

- [jira](https://codedotorg.atlassian.net/browse/STAR-161)

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/applab/designMode.js [hunk 1]
```diff
@@ -383,6 +385,8 @@ designMode.updateProperty = function(
 
       if (ICON_PREFIX_REGEX.test(value)) {
         element.src = assetPrefix.renderIconToString(value, element);
+      } else if (DATA_PREFIX_REGEX.test(value)) {
+        element.src = assetPrefix.fixPath(value);
```
Comment (jmkulwik) [inline apps/src/applab/designMode.js]: Question: Can you give a little context on why you added fixPath here?

File: apps/src/assetManagement/assetPrefix.js [hunk 2]
```diff
@@ -13,6 +13,8 @@ const ABSOLUTE_CDO_CURRICULUM_REGEXP = new RegExp(
   'i'
 );
 
+export const DATA_URL_PREFIX_REGEX = new RegExp('^data:image');
```
Comment (maddiedierker) [inline apps/src/assetManagement/assetPrefix.js:16]: super nit: we include the `/` in our other regex patterns below (e.g., `'icon://'` on line 18) -- should we do the same here for consistency?

Comment (maddiedierker) [reply apps/src/assetManagement/assetPrefix.js:16]: hmmmm, nevermind, it looks like the forward slash _may_ be part of the mime type (like `data:image/gif`), but isn't always present. so don't add that forward slash!

**Review by maddiedierker — APPROVED** [review]

File: apps/src/applab/designMode.js (hunk 1 above)
Comment (epeach) [reply apps/src/applab/designMode.js]: Not needed. Removed in latest commit.

**Review by jmkulwik — APPROVED** [review]
LGTM!

**Pull request merged**